Formula/fgsl.rb

```diff
@@ -0,0 +1,30 @@
+class Fgsl < Formula
+  desc "Fortran bindings for the GNU Scientific Library"
+  homepage "https://www.lrz.de/services/software/mathematik/gsl/fortran/"
+  url "https://www.lrz.de/services/software/mathematik/gsl/fortran/download/fgsl-1.2.0.tar.gz"
+  sha256 "00fd467af2bb778e8d15ac8c27ddc7b9024bb8fa2f950a868d9d24b6086e5ca7"
+
+  depends_on "pkg-config" => :build
+  depends_on "gcc"
+  depends_on :fortran
+  depends_on "gsl"
+
+  def install
+    ENV.deparallelize
+
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "CC=gcc-7",
+                          "--prefix=#{prefix}"
+    system "make"
+    system "make", "check"
```

Comment:
This line should be removed as we only run `make check` and `make test` for security software.